static/app/views/projectInstall/issueAlertNotificationContext.tsx

```diff
@@ -0,0 +1,32 @@
+import {createContext, useContext} from 'react';
+
+import type {IssueAlertActionType} from 'sentry/types/alerts';
+import type {OrganizationIntegration} from 'sentry/types/integrations';
+
+export type IssueAlertNotificationContextValue = {
+  alertNotificationAction: IssueAlertActionType[];
+  alertNotificationChannel: string | undefined;
+  alertNotificationIntegration: OrganizationIntegration | undefined;
+  alertNotificationProvider: string | undefined;
+  setAlertNotificationAction: (method: IssueAlertActionType[]) => void;
+  setAlertNotificationChannel: (channel: string | undefined) => void;
+  setAlertNotificationIntegration: (
+    integration: OrganizationIntegration | undefined
+  ) => void;
+  setAlertNotificationProvider: (provider: string | undefined) => void;
```

Comment:
I think we can probably get rid of this context, but just for the future, I don't think these keys need to be nearly this verbose. They are already contained inside a context called `IssueAlertNotificationContext` so it would be easier to read these by just stripping off the `alertNotification` part.